server/services/conditionsService.ts

```diff
@@ -75,8 +76,10 @@ export class ConditionsService {
     }
   }
 
-  getStandardConditions() {
-    return standardConditions
+  getStandardConditionsByVersion(standardConditionsVersion: StandardConditionsVersion) {
+    if (standardConditionsVersion == 1) return standardConditionsV1
```

Comment:
This line is technically redundant due to the default being V1, but thought the clarity of it was more important